package.json

```diff
@@ -29,7 +29,7 @@
     "format": "prettier --ignore-path .eslintignore --write ./ && npm run lint:fix",
     "format:deno": "deno fmt integration/helpers/deno-template packages/remix-deno templates/deno --ignore=packages/remix-deno/node_modules",
     "license": "node ./scripts/license.js",
-    "lint": "eslint --cache --ext .tsx,.ts,.js,.jsx,.md .",
```

Comment:
Aren't we loosing linting JS/TS code in MD-files now? 🤔

Reply:
ts is enabled via typescript-eslint https://typescript-eslint.io/architecture/parser#extrafileextensions, but i need to re-add md

Reply:
actually, md is enabled by https://github.com/eslint/eslint-plugin-markdown#configuring and in eslint v7, that is enough https://github.com/eslint/eslint-plugin-markdown#eslint-v7